Gemfile

```diff
@@ -1,5 +1,7 @@
 source 'https://rubygems.org'
 
+gem 'deface'
```

Comment:
@PSzyszkaSpark `deface` dependency needs to be added to the gemspec file and required here: https://github.com/spree-contrib/spree_multi_currency/blob/master/lib/spree_multi_currency.rb